test/unit/org/apache/cassandra/index/sai/disk/v1/kdtree/BKDReaderTest.java

```diff
@@ -402,39 +400,14 @@ private BKDReader finishAndOpenReaderOneDim(int maxPointsPerLeaf, BKDTreeRamBuff
                                                                  maxPointsPerLeaf,
                                                                  Integer.BYTES,
                                                                  Math.toIntExact(buffer.numRows()),
-                                                                 buffer.numRows(),
+                                                                 buffer.numPoints(),
                                                                  new IndexWriterConfig("test", 2, 8));
 
         final SegmentMetadata.ComponentMetadataMap metadata = writer.writeAll(buffer.asPointValues());
         final long bkdPosition = metadata.get(IndexComponentType.KD_TREE).root;
-        assertThat(bkdPosition, is(greaterThan(0L)));
+        assertTrue(bkdPosition > 0L);
```

Comment:
Why this change? 
Doesn't look relevant to the actual fix, and I think that makes error reporting slightly worse. Assertj fluent assertions should give more descriptive messages than the standard unit assertions.

Reply:
This method is deprecated, so using it doesn't seem to be a good idea. I made a fix, which is suggested in the deprecation documentation. So the error will be still meaningful and no deprecation warning.